File: backend-management-service/src/main/java/com/synexis/management_service/dto/response/RatingResponse.java
package com.synexis.management_service.dto.response;

import lombok.*;

import java.time.LocalDateTime;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class RatingResponse {

    private Long id;
    private Long serviceId;
    private Long clientId;
    private Long partnerId;
    private Integer score;
    private String comment;
    private LocalDateTime createdAt;

}

File: mobile-partner/app/src/main/java/com/sismptm/partner/data/remote/api/dto/RatingResponse.kt
package com.sismptm.partner.data.remote.api.dto

/**
 * Response DTO for a rating.
 */
data class RatingResponse(
    val id: Long,
    val serviceId: Long,
    val clientId: Long,
    val partnerId: Long,
    val score: Int,
    val comment: String?,
    val createdAt: String?
)
